### PR TITLE
refactor in address and addres-detail module

### PR DESCRIPTION
**Pull request by drcoolsanjeev**
### Type of change

- Refactoring

### Description

- Removed some inline styles
- created a enum for addressTypes to avoid using string
- Break Create address configuration into component and container

### CODE REVIEW

File: console/console-init/ui/src/modules/address/dialogs/CreateAddress/Configuration.tsx [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+  const { loading, error, data } = useQuery<IAddressTypes>(
+    RETURN_ADDRESS_TYPES,
+    {
+      variables: {
+        a: addressSpaceType
+      }
+    }
+  );
+  if (loading) return <Loading />;
+  if (error) return <Loading />;
```
Comment (anukritijha) [inline console/console-init/ui/src/modules/address/dialogs/CreateAddress/Configuration.tsx]: I don't think we need a condition for errors here anymore

File: console/console-init/ui/src/modules/address/dialogs/CreateAddress/Configuration.tsx [hunk 2]
```diff
@@ -0,0 +1,191 @@
+/*
```
Comment (anukritijha) [inline console/console-init/ui/src/modules/address/dialogs/CreateAddress/Configuration.tsx]: Can we qualify this as a container, as it's mainly calling graphQL for different scenarios?

File: console/console-init/ui/src/modules/address/dialogs/CreateAddress/Configuration.tsx (hunk 1 above)
Comment (drcoolsanjeev) [reply console/console-init/ui/src/modules/address/dialogs/CreateAddress/Configuration.tsx]: I will remove this in next PR.

File: console/console-init/ui/src/modules/address/dialogs/CreateAddress/Configuration.tsx (hunk 2 above)
Comment (drcoolsanjeev) [reply console/console-init/ui/src/modules/address/dialogs/CreateAddress/Configuration.tsx]: Yeah , but we discussed to put all modals like create and edit in dialogs folder

**Pull request merged**